packages/stylelint-plugin/src/utils/checkImportExists.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2020 Palantir Technologies, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+import type { Root } from "postcss";
+
+/**
+ * Returns true if the given import exists in the file, otherwise returns false.
+ */
+export function checkImportExists(root: Root, importPath: string): boolean {
+    let hasBpVarsImport = false;
+    root.walkAtRules(/^import$/i, atRule => {
```

Comment:
does the AST at this point have stripped off the `@` from `@import "~@blueprintjs/core/lib/scss/variables";` ?

Reply:
Yes